### PR TITLE
Task/upgrade atlassian stack

### PR DESCRIPTION
**Pull request by victorpablosceruelo**
Modifications needed to be able to upgrade atlassian suite without reinstalling everything.

Basically I've included a function "atlassian_stack_reset" that removes the atlassian suite and reinstalls it.

[openshift@ip-10-0-0-50 ods-core]$ ods-devenv/scripts/deploy.sh --branch task/upgrade-atlassian-stack --target atlassian_stack_reset

I've used it to upgrade my box from jira 8.5.8 to jira 8.20.5.

The current license is not valid, but this is a minor issue, since we can enter it manually.

### CODE REVIEW

**Comment by stitakis** [comment]
@victorpablosceruelo the PR looks good. Currently the ODS build is broken. I'll approve this PR once we get the build running successful again.

**Comment by victorpablosceruelo** [comment]
@stitakis, you mean the comments here?

https://github.com/opendevstack/ods-quickstarters/issues/706

**Comment by stitakis** [comment]
@victorpablosceruelo yes

**Review by stitakis — DISMISSED** [review]

File: .github/workflows/continuous-integration-workflow.yml [hunk 1]
```diff
@@ -107,20 +107,20 @@ jobs:
...
+        name: Setup Go 1.18
+        uses: actions/setup-go@v3
         with:
-          go-version: 1.12
+          go-version: 1.18
       -
         name: Download goimports
         run: |
-          go get golang.org/x/tools/cmd/goimports
+          go install golang.org/x/tools/cmd/goimports@latest
```
Comment (stitakis) [inline .github/workflows/continuous-integration-workflow.yml:119]: just a comment... depending on `latest` might make this to break in future

**Review by stitakis — APPROVED** [review]

**Pull request merged**